src/app/src/main/java/com/uphold/uphold_android_sdk/model/card/Address.java

```diff
@@ -43,4 +45,14 @@ public String getNetwork() {
         return network;
     }
 
+    /**
+     * Gets the tag if available.
```

Comment:
Just `Gets the tag.`?

Reply:
I added that because the field is optional and it will be available only for XRP.